docs/streams/developer-guide/dsl-api.html

```diff
@@ -3428,6 +3428,17 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
             <pre class="brush: scala;">
               libraryDependencies += "org.apache.kafka" %% "kafka-streams-scala" % "{{fullDotVersion}}"
             </pre>
+            <p>
+              <strong>Note</strong>:
+              <p>
+                There is an upstream dependency that causes trouble in SBT builds. This problem is fixed in <code>2.1.1</code>, and <code>2.2.0</code>.
```

Comment:
Neither 2.1.1 nor 2.2.0 are available yet.

Reply:
True, but they should be released soon, right? It seems temporally complicated to come back and add this statement in a month. On the other hand, I was assuming that it would be pretty obvious what's going on in the mean time.

Reply:
\cc @guozhangwang WDYT?

Reply:
I think we can add this here: the next time `docs` folder will be copied would be 2.2.0 release.